middlewares/isValidId.js

```diff
@@ -0,0 +1,10 @@
+import { isValidObjectId } from 'mongoose';
+import { HttpError } from '../helpers/HttpError.js';
+
+export const isValidId = (req, res, next) => {
+  const { id } = req.params;
```

Comment:
be careful with this, because you don't have `id` like this everywhere, you can have `reviewId`, `chefId`, `dishId` etc.